src/js/components/ToggleGroup/stories/multiple.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Box, ToggleGroup } from 'grommet';
+
+export const Multiple = () => (
+  <Box pad="large">
+    <ToggleGroup options={['Bold', 'Italic', 'Underline']} multiple />
```

Comment:
These should be icons, not text.

Please check that the screenreader announces these properly.

Reply:
Done.